damage.py

```diff
@@ -1,5 +1,31 @@
 # TODO: it seems that hp are ceiled while damage floored. (however it seems that the ad is rounded in the stat section ingame)
 
+def pre_mitigation_physical_damage(base_attack_damage: float, bonus_attack_damage: float, damage_modifier_flat: float,
+                                   damage_modifier_percent_mult_factor: float, crit: 'bool', crit_damage: float):
+    """
+    Calculates the pre-mitigation physical damage of a spell or an autoattack
+    All values regarding damage modifiers should include the buffs/debuffs coming from spells, summoner spells, or items
+    from both the attacker AND the defender
+    """
+    if crit:
+        return (base_attack_damage + bonus_attack_damage) * damage_modifier_percent_mult_factor * (
+                    1.75 + crit_damage) + damage_modifier_flat
+    return (base_attack_damage + bonus_attack_damage) * damage_modifier_percent_mult_factor + damage_modifier_flat
```

Comment:
this is a special case of `avg_pre_mitigation_physical_damage` with `crit_chance` set to either 0 or 1.